mozilla/blurts-server
# Don’t always redirect to /user/dashboard after auto sign-in

# References: 
Jira: [MNTOR-2820](https://mozilla-hub.atlassian.net/browse/MNTOR-2820)



# Description

Send users to the requested route after authentication that has been triggered by `AutoSignIn`.

# How to test

1. Visit `/` and make sure you are not authenticated.
2. Navigate to a route that triggers auto sign-in: Enter for example `/user/dashboard/fix` into the address bar.
3. You should not get redirected to `/dashboard` anymore.

[MNTOR-2820]: https://mozilla-hub.atlassian.net/browse/MNTOR-2820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
